### PR TITLE
added marks_size column in system.parts

### PR DESCRIPTION
**Pull request by KochetovNicolai**

### CODE REVIEW

File: dbms/src/Storages/System/StorageSystemParts.cpp [hunk 1]
```diff
@@ -7,6 +8,7 @@
 #include <Storages/System/StorageSystemParts.h>
 #include <Storages/StorageMergeTree.h>
 #include <Storages/StorageReplicatedMergeTree.h>
+#include <Storages/StorageLog.h>
```
Comment (alexey-milovidov) [inline dbms/src/Storages/System/StorageSystemParts.cpp]: Double include.

File: dbms/src/Storages/System/StorageSystemParts.cpp [hunk 2]
```diff
@@ -1,3 +1,4 @@
...
```
Comment (alexey-milovidov) [inline dbms/src/Storages/System/StorageSystemParts.cpp]: This is not the right one. Look at `MarkInCompressedFile.h`.
(You could consider the implementation of StorageMergeTree.)

**Review by alexey-milovidov — CHANGES_REQUESTED** [review]

File: dbms/src/Storages/System/StorageSystemParts.cpp [hunk 3]
```diff
@@ -213,6 +216,7 @@ BlockInputStreams StorageSystemParts::read(
             block.getByPosition(i++).column->insert(part->name);
             block.getByPosition(i++).column->insert(static_cast<UInt64>(active_parts.count(part)));
             block.getByPosition(i++).column->insert(part->size);
+            block.getByPosition(i++).column->insert(part->size * part->columns.size() * sizeof(Mark));
```
Comment (alexey-milovidov) [inline dbms/src/Storages/System/StorageSystemParts.cpp]: Also look at `MergeTreeDataPartChecksums` and it's usage across the codebase.

Comment (KochetovNicolai) [inline dbms/src/Storages/System/StorageSystemParts.cpp]: Do you mean that I need to add checksums size for primary.idx file?

**Pull request merged**